odoo/addons/test_inherit/tests/test_inherit.py

```diff
@@ -87,6 +88,51 @@ def test_51_define_model_inherit(self):
         model = self.env['test_inherit_parent']
         self.assertEqual(bool(model._fields['foo']), True)
 
+    def test_50_define_model_with_mixin(self):
+        class MixinFoo:
+            def foo(self):
+                pass
+
+        class MixinBar:
+            def bar(self):
+                pass
+
+        class NewTestInheritModel(MixinFoo, models.AbstractModel, MixinBar):
```

Comment:
I thought that using `MixinBar` which does not inherit at all from `BaseModel` (even indirectly), in a model inheritance would raise a TypeError.